src/util/helpers.js

```diff
@@ -35,12 +34,10 @@ const analysisFields = () => {
         'trackedEntityType[id,displayName~rename(name)]',
         'organisationUnitSelectionMode',
         '!href',
-        '!publicAccess',
```

Comment:
I considered whether we should leave these alone (lines 38, 43, 52) since removing them results in these fields now being returned in the 2.40 maps response. But they are obviously not in use since they're currently being removed, and when saving a map, they are not included in the payload (see change in helpers file)

Reply:
Decided to leave these exclusions in until v40 is no longer in support.